docker-compose.yml

```diff
@@ -146,6 +146,17 @@ services:
       mode: replicated
       replicas: 1
 
+  dsv:
+    # listens internally on port 3010 by default
+    image: "quay.io/duodecim/ebmeds-diagnosis-specific-view:${EBMEDS_VERSION}"
```

Comment:
Should the image name be `ebmeds-dso`?

Reply:
That'd be cleaner, sure.

Reply:
https://jira.duodecim.fi/browse/EBMEDS-1309 Unify naming scheme for DSO ticket created.